### PR TITLE
[nobug, build] - Updated BR yml and removed unused workflows

### PR DESCRIPTION
**Pull request by nbhasin2**
Also, updated all machines to latest BR elite g2 which are faster and use same credits (2) as standard machines

Build time for release build was about 1.5min faster. There are other machines as well that we can use but this is a good start. 

Follow-up: Change the machines accordingly as there are plenty to pick from.

### CODE REVIEW

**Comment by nbhasin2** [comment]
I have marked this PR as Do not merge as this has some workflow changes along with machine upgrades. 

@isabelrios and @AaronMT tagged you both for review before merging.

**Review by isabelrios — APPROVED** [review]
LGTM, thanks for cleaning this up.

File: bitrise.yml
Comment (isabelrios) [inline bitrise.yml:395]: This workflow can be removed to if you want to do it in this PR if not we will do it later in a follow up. Thanks!

**Review by AaronMT — APPROVED** [review]

File: bitrise.yml
Comment (nbhasin2) [reply bitrise.yml]: @isabelrios I removed this as well :)

**Comment by isabelrios** [comment]
Closing/reopening to re-trigger Bitrise to get a green check

**Comment by isabelrios** [comment]
There is a failure with this test:` PhotonActionSheetTest.testPinToTop()` I think due to a timing issue trying to tap on PAM before it is ready... if it blocks from merging, please feel free to disable it.

**Comment by nbhasin2** [comment]
Merging this @isabelrios as I don't believe any issues with BR script will cause above test to fail.

**Pull request merged**
